### ISSUE TITLE
collect-stack-trace is not a recognized LoStanza primitive.

### ISSUE THREAD

**Issue by SCOTT-HAMILTON** [StanzaOrg/lbstanza-old#141]
I made a Nix derivation to build lbstanza from source, 
cf https://github.com/SCOTT-HAMILTON/nur-packages/blob/master/pkgs/lbstanza-bin/default.nix
and https://github.com/SCOTT-HAMILTON/nur-packages/blob/master/pkgs/lbstanza/default.nix

It was building version 1.14.33 and above successfully but since [this commit](https://github.com/stanzaOrg/lbstanza/commit/1d96c0294011a48d6bb8f9737bd4a10e73838289) I get this error message (found from bisecting)
```
  Building Stanza for Linux
  Cleaning Stanza files
  Compiling Linux Stanza Pkgs
  core/core.stanza:300.28: collect-stack-trace is not a recognized LoStanza primitive.
```

**Comment by CuppoJava** [comment]
Hi Scott.

I made some bad commits in the recent sprint. (That was one of them.)

The tag `0.15.0` should get past the issue. But keep in mind that you will need a binary for Stanza `0.15.0` to build this version. (This happens rarely, where you need a newer compiler to build a newer commit, but it happens from time to time.)

**Comment by SCOTT-HAMILTON** [comment]
Maybe a github workflow CI would be better than me openning an issue each time master doesn't build. I can make a PR if needed.

**Comment by CuppoJava** [comment]
Yeah! Would sincerely appreciate that. I haven't had the time to get caught up with Github's flows.

**Comment by SCOTT-HAMILTON** [comment]
Here it is https://github.com/StanzaOrg/lbstanza/pull/142
